src/main/java/mate/academy/spring/dto/request/MovieRequestDto.java

```diff
@@ -1,6 +1,11 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
 public class MovieRequestDto {
+    @NotNull
+    @Min(1)
```

Comment:
What the annotation @Min validate above String field?

Reply:
replaced with size